code/stable/dblpendulum/SRS/DblPendulum_SRS.tex

```diff
@@ -1104,9 +1104,9 @@ \subsubsection{Properties of a Correct Solution}
 \\
 \midrule
 \endhead
-$α$ & $α\gt{}0$
+$α$ & ${α\gt{}0}$
 \\
-${θ_{p}}$ & ${θ_{p}}\gt{}0$
+${θ_{p}}$ & ${{θ_{p}}\gt{}0}$
```

Comment:
I'm not yet sure what's causing this double {}, but it's only appearing in the TeX version as of right now. I think that current changes wrap things in a PrintingAST Row twice somehow, but I haven't figured it out yet.

Reply:
Fixed